modules/minimum-ibm-power-system-requirements.adoc

```diff
@@ -9,7 +9,12 @@
 
 You can install {product-title} version {product-version} on the following IBM hardware:
 
-* IBM Power8, Power9, or Power10 processor-based systems
+* IBM Power9 or Power10 processor-based systems
+
+[NOTE]
+====
+Support for {op-system} functionality for IBM POWER8 all models, IBM POWER9 AC922, IBM POWER9 IC922, and IBM POWER9 LC922 is deprecated. These hardware models remain fully supported in {product-title} 4.12. However, Red Hat recommends that you use later hardware models.
```

Comment:
Could we maybe slightly revise this note? It's a little confusing as-is. Maybe something like...

Support for {op-system} functionality for all IBM POWER8 models, IBM POWER9 AC922, IBM POWER9 IC922, and IBM POWER9 LC922, is deprecated. These hardware models remain fully supported in {product-title} 4.12. However, Red Hat recommends that you use later hardware models.

Reply:
Ack